README.md

```diff
@@ -45,6 +47,21 @@ The following instructions have been tested on Linux.
 
 Set up the Bot locally using docker-compose. Best if you want to quickly try it out.
 
+### Beta Testnet (beta.polkabtc.io)
+
+```shell
+git clone https://github.com/interlay/bridge-bot
+cd bridge-bot
+yarn install
+source .env.testnet
```

Comment:
I think there should be a `set -o allexport` before `source .env.testnet`. The variables in `.env.testnet` are declared without `EXPORT` so they can be set in the systemd service environment

Reply:
added, thanks